larq_compute_engine/core/bgemm_kernels_arm64.h

```diff
@@ -552,12 +552,12 @@ void CheckOffsetsInKernelParams64BP(const Params&) {
 // During accumulation, v0 -- v3 are used to load data from LHS and
 // v4 -- v7 from RHS:
 //
-//                                      int32 RHS 2x4 block
+//                                    int64 RHS 2x4 block
 //                          /--------------------------------------\
 //                          |v4.d[0]         ...          v7.d[0]  |
 //                          |v4.d[1]         ...          v7.d[1]  |
 //                          \--------------------------------------/
-//    int32 LHS 4x2 block
+//     int64 LHS 4x2 block
```

Comment:
@arashb/@tombana I could be wrong about this but I think these are int64 blocks (here and line 555)?

Reply:
you are right, it should be int64.